hack/virtc/README.md

```diff
@@ -51,6 +51,28 @@ $ sudo su
 
 `virtcontainers-setup.sh` setup your environment performing different tasks. Particularly, it creates a __busybox__ bundle, and it creates CNI configuration files needed to run `virtc` with CNI plugins.
 
+### Get cc-proxy (optional)
+
+If you plan to start virtc with hyperstart agent, you will have to use [cc-proxy](https://github.com/clearcontainers/proxy) as a proxy, meaning you have to perform extra steps to setup your environment.
+
+```
+$ go get github.com/clearcontainers/proxy
+$ make
+$ sudo make install
+```
+If you want to see the traces from the proxy when virtc will run, you can manually start it with appropriate debug level:
+
+```
+$ sudo su
+# /usr/libexec/clearcontainers/cc-proxy -v 3
+```
+This should generate the following output:
```

Comment:
Pedantic I know, but the user won't get that *exact* output, so I'd say:

    This will generate output similar to the following: